### ISSUE TITLE
Setting several jobs in single GPU with different seeds

### ISSUE THREAD

**Issue by zuoxingdong** [pytorch/pytorch#12567]
When the model is small and a lot of hyperparameter search needed, then train the model with different random seeds on a single GPU. 

e.g.
```python
# in job1
device = torch.device('cuda:3')
torch.manual_seed(1)

# in job2
device = torch.device('cuda:3')
torch.manual_seed(2)

# in job3
device = torch.device('cuda:3')
torch.manual_seed(3)
```
and all 3 jobs run in parallel on 4th GPU. Because we know that `manual_seed` also seed CUDA device. I am confused how large impact the randomness interference ?

**Comment by ngimel** [comment]
Closing due to age
